settings.d/dynflow.yml.example

```diff
@@ -1,9 +1,3 @@
 ---
 :enabled: true
 :database: /var/lib/foreman-proxy/dynflow/dynflow.sqlite
-:core_url: 'http://127.0.0.1:8008'
```

Comment:
should we add `execution_plan_cleaner_age` and `console_auth` here?
Also, will the current settings get migrated somehow, or will that need to be handled manually by user?